ocaml/xapi/helpers.ml

```diff
@@ -1048,19 +965,6 @@ let find_health_check_task ~__context ~sr =
       Eq (Field "name__description", Literal (Ref.string_of sr))
     ))
 
-(* Copy the snapshot metadata from [src_record] to the VM whose reference is [dst_ref]. *)
-(* If a lookup table is provided, then the field 'snapshot_of' is translated using this *)
-(* lookup table. *)
-let copy_snapshot_metadata rpc session_id ?lookup_table ~src_record ~dst_ref =
```

Comment:
I'd like to know the history of this; when it came in and when it became dead code.

Reply:
It comes in by the Initial import from hg on 2009, but it's hard to track when it became dead.

Reply:
looks like it was here: 8e57c84c647076a2f6190ada0652f531796fdf90

Reply:
OK, so it's not recent :-) I'm happy.